pkg/drift/drift.go

```diff
@@ -42,25 +45,25 @@ func Process(ctx context.Context, organizationID, bucketQuery string) error {
 	if err != nil {
 		return fmt.Errorf("failed to determine terraform state GCS buckets: %w", err)
 	}
-	log.Printf("Fetching IAM for 1 Org, %d Folders and %d Projects\n", len(folders), len(projects))
+	logger.Debugw("Fetching IAM for 1 Org, %d Folders and %d Projects\n", len(folders), len(projects))
 
 	gcpIAM, err := actualGCPIAM(ctx, organizationID, folders, projects)
 	if err != nil {
 		return fmt.Errorf("failed to determine GCP IAM: %w", err)
 	}
-	log.Printf("Fetching terraform state from %d Buckets\n", len(buckets))
+	logger.Debugw("Fetching terraform state from %d Buckets\n", len(buckets))
 	tfIAM, err := terraformStateIAM(ctx, organizationID, folders, projects, buckets)
 	if err != nil {
 		return fmt.Errorf("failed to parse IAM from Terraform State: %w", err)
 	}
-	log.Printf("Found %d gcp IAM entries\n", len(gcpIAM))
-	log.Printf("Found %d terraform IAM entries\n", len(tfIAM))
+	logger.Debugw("Found %d gcp IAM entries\n", len(gcpIAM))
```

Comment:
```suggestion
	logger.Debugw("found gcp iam entries", "entries", gcpIAM)
```

Reply:
(etc). This will generate structured (JSON) logs in prod and KEY=VALUE logs in dev.

Reply:
Done